public/index.html

```diff
@@ -30,6 +30,7 @@
       content="/images/favicon/browserconfig.xml"
     />
     <meta name="theme-color" content="#ffffff" />
+    <meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=1.0, minimum-scale=0.86">
```

Comment:
I think having `maximum-scale=1.0` will prevent mobile users from pinch zoom? If so, this will be a concern for accessibility.

Reply:
Hmm I'm not entirely sure, but there is a flag to allow users to pinch zoom. Looking through some examples a lot of the time people put it to 5. I'll change it to five and keep the other values the same. This change doesn't actually make any visual difference when the user first loads the page.